examples/action.py

```diff
@@ -40,13 +41,10 @@
         await run_action(unit)
 
     await model.disconnect()
-    model.loop.stop()
 
 
-logging.basicConfig(level=logging.DEBUG)
-ws_logger = logging.getLogger('websockets.protocol')
-ws_logger.setLevel(logging.INFO)
-loop = asyncio.get_event_loop()
-loop.set_debug(False)
-loop.create_task(run())
-loop.run_forever()
+if __name__ == '__main__':
+    logging.basicConfig(level=logging.DEBUG)
+    ws_logger = logging.getLogger('websockets.protocol')
+    ws_logger.setLevel(logging.INFO)
```

Comment:
@johnsca I actually had the helper half written. The problem is that the values we set in the examples aren't really values that you'd set for production. I wound up backing it out, and thinking maybe I'd add it in another pass.